en/installation-guide/database-initialization.rst

```diff
@@ -58,49 +63,47 @@ Restore the latest data model that also includes demo data:
 
   .. figure:: images/demodata-restore.jpg
 
-  * Check the `Restore Options` Tab
+  * Click the `Restore Options` Tab and check these options
+  
 
   .. figure:: images/demodata-restore_options.jpg
 
   * Click `Restore`
   
   * Check whether in Message window `Restoring backup on the server` is successfully completed.
   
-  .. note:
+.. note::
   
-    If the the Restore is failed and the Detail reads something like
-  "pg_restore: [archiver] unsupported version (1.13) in file header"
-  or in German "pg_restore: [Archivierer] nicht unterstützte Version (1.13) im Dateikopf"
-  try updating your PostgreSQL, see https://stackoverflow.com/questions/49064209/getting-archiver-unsupported-version-1-13-in-file-header-when-running-pg-r
+   If the the Restore is failed and the Detail reads something like "pg_restore: [archiver] unsupported version (1.13) in file header" or in German "pg_restore: [Archivierer] nicht unterstützte Version (1.13) im Dateikopf" try updating your PostgreSQL, see https://stackoverflow.com/questions/49064209/getting-archiver-unsupported-version-1-13-in-file-header-when-running-pg-r
```

Comment:
twice the - eliminate If the the by "If the Restore fails and the ...

Reply:
done